modules/cef/event-format-parser.c

```diff
@@ -264,6 +304,48 @@ _extract_msg_expr(FilterXFunctionArgs *args, GError **error)
   return msg_expr;
 }
 
+static gboolean
+_extract_optional_args(FilterXFunctionEventFormatParser *self, FilterXFunctionArgs *args, GError **error)
+{
+  gboolean exists;
+  gsize len;
+  const gchar *value;
+
+  value = filterx_function_args_get_named_literal_string(args, EVENT_FORMAT_PARSER_ARG_NAME_PAIR_SEPARATOR, &len,
+                                                         &exists);
+  if (exists)
+    {
+      if (len < 1 || !value)
+        {
+          g_set_error(error, FILTERX_FUNCTION_ERROR, FILTERX_FUNCTION_ERROR_CTOR_FAIL,
+                      EVENT_FORMAT_PARSER_ERR_EMPTY_STRING, EVENT_FORMAT_PARSER_ARG_NAME_PAIR_SEPARATOR);
+          goto error;
+        }
+      if (len > EVENT_FORMAT_PARSER_PAIR_SEPARATOR_MAX_LEN)
```

Comment:
`max_len - 1` is what we can accept, as `max_len` is the buf size, and we need one character for NUL termination, right?